src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/JavaScriptEncoder.cs

```diff
@@ -23,6 +23,15 @@ public static JavaScriptEncoder Default
             get { return DefaultJavaScriptEncoder.Singleton; }
         }
 
+        /// <summary>
+        /// Returns a built-in instance of <see cref="JavaScriptEncoder"/> that is less strict about what gets encoded.
+        /// </summary>
+        /// <remarks>TODO to explain nuance of what it does, when it is safe to use it, and implications user is opting into (why is it unsafe).</remarks> 
```

Comment:
Need help with the docs here to encapsulate the "new behavior" as specified in the proposed issue: https://github.com/dotnet/corefx/issues/39410#issue-467106665

Reply:
Done.

Reply:
Based on offline discussions, it would be good to have more detailed docs (with code samples) providing guidance on when you can and should not use this static.